test/strategies/rust.ts

```diff
@@ -147,6 +153,7 @@ describe('Rust', () => {
       const updates = release!.updates;
       assertHasUpdate(updates, 'crates/crate1/Cargo.toml', CargoToml);
       assertHasUpdate(updates, 'crates/crate2/Cargo.toml', CargoToml);
+      assertHasUpdate(updates, 'Cargo.toml', CargoToml);
```

Comment:
Is this the main regression here, that the root Cargo.toml wasn't updated?

Reply:
Yeah, this was the initial failing test written to capture the bug